packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/ExpressionTypeProviderWriter.cs

```diff
@@ -18,8 +18,7 @@ public ExpressionTypeProviderWriter(CodeWriter writer, TypeProvider provider)
 
         public virtual void Write()
         {
-            var ns = TypeProvider.GetDefaultModelNamespace(CodeModelPlugin.Instance.Configuration.Namespace);
```

Comment:
it is incorrect that we always write the namespace for models here.
The namespace should come from the provider or the type that the provider produces.